pyproject.toml

```
@@ -1,18 +1,21 @@
-[tool.poetry]
+[project]
 name = "betterproto"
 version = "2.0.0b7"
 description = "A better Protobuf / gRPC generator & library"
-authors = ["Daniel G. Taylor <danielgtaylor@gmail.com>"]
+authors = [
+    {name = "Daniel G. Taylor", email = "danielgtaylor@gmail.com"}
+]
 readme = "README.md"
 repository = "https://github.com/danielgtaylor/python-betterproto"
 keywords = ["protobuf", "gRPC"]
 license = "MIT"
 packages = [
     { include = "betterproto", from = "src" }
 ]
+requires-python = ">=3.8"
```

```suggestion
requires-python = ">=3.8,<4.0"
```

For equivalency with previous configuration.

---

I think this would be unnecessarily strict, there's no reason to assume 4.0 will break compatibility at this time, generally upper limits for python versions should only be used when there's a known incompatibility AFAIU.

---

Debatable. But for now my comment is that the prior version was `^3.8` which has an upper bound of `<4.0`.

---

applied as suggested